templates/docs/layouts/fluid-breakout.md

```diff
@@ -0,0 +1,103 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Breaking out of the grid | Layouts
+---
+
+## Breaking out of the grid
+
+<hr>
+
+In some cases, there might be a good reason to break out of the constraints of a 12 column grid and allow content to bleed into the page margins - for example, tables with many columns that would otherwise result in heavy truncation or wrapping, charts with a lot of detail along the x axis, or card layouts that aim to impress with the abundance of available content. The fluid breakout layout allows you to do this.
+
+### Anatomy of the fluid breakout layout
+
+#### Wrapper
+
+The `.l-fluid-breakout` serves as a wrapper, and allows for an aside and a main area. On screens smaller than `$breakpoint--large`, it acts as a single column. On larger screens it switches to a 3 column layout - a central area and two aside areas on either side.
+
+The wrapper aims to align as much as possible with the 12 column grid. On smaller screens, than is limited to ensuring the padding of both layouts match. Once the window width becomes large enough, the left edges of the regular 12 column grid and the central column of the fluid breakout layout also align. For this to be possible, two conditions need to be met:
+
+- <p>the central area's width needs to match the 12 column grid's width (we achieve this by setting the central area's max-width to the same value as the 12 column grid's max-width)</p>
```

Comment:
Why is this using paragraphs inside the list?

Reply:
not needed in narrow layouts, I think I added it when I experimented with the wider area - as list items have no max width. Removing as not currently needed.